pkgs/development/python-modules/trogon/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, poetry-core
+, click
+, textual
+}:
+
+buildPythonPackage rec {
+  pname = "trogon";
+  version = "0.5.0";
+  pyproject = true;
+
```

Comment:
The package should also indicate that it requires at least Python 3.7 (e.g. through `pythonOlder`).